core/jazz_t-vault/test/test.js

```diff
@@ -15,13 +15,3519 @@
 // =========================================================================
 
 const assert = require('chai').assert;
+const expect = require('chai').expect;
+const sinon = require('sinon');
+const awsContext = require('aws-lambda-mock-context');
 const index = require('../index');
+const configModule = require("../components/config.js");
+const vault = require("../components/utils/vault.js");
+const validations = require("../components/utils/validations.js");
 
-describe('Sample', () => {
-  it('tests handler', (done) => {
+//Validations
+describe('Validations', () => {
+  beforeEach(function () {
+    event = {
+      "method": "POST",
+      "stage": "test",
+      "resourcePath": "test/service",
+      "body": {
+        "name": "tester",
+        "owner": "test",
+        "description": "testsafe"
+      }
+    };
+    context = awsContext();
+    callback = (value) => {
+      return value;
+    };
+    config = configModule.getConfig(event, context);
+  });
+
+  it('should resolve while validating create safe input params with valid input', (done) => {
+    validations.validateCreateSafeInput(event)
+      .then((result) => {
+        assert(true);
+      })
+    done();
+  });
+
+  it('should reject while validating create safe input params with empty input', (done) => {
+    event.body = {};
+    validations.validateCreateSafeInput(event)
+      .catch((err) => {
+        expect(err).to.include({
+          errorType: 'inputError',
+          message: 'Input cannot be empty'
+        });
+      });
+    done();
+  });
+
+  it('should reject while validating create safe input params with out event.body', (done) => {
```

Comment:
nitpick - with out is a single word `without` - several instances.

Reply:
Done